Doc/library/os.rst

```diff
@@ -3578,6 +3579,11 @@ written in Python, such as a mail server's external command delivery program.
    process.  On Windows, the process id will actually be the process handle, so can
    be used with the :func:`waitpid` function.
 
+   On VxWorks, if *mode* is :const:`P_NOWAIT`, this function returns the process id
+   of the new process; if *mode* is :const:`P_WAIT`, only returns the new process's
+   exit code if it exits normally. The process id will actually be the process
+   handle, so can be used with the :func:`waitpid` function.
```

Comment:
This description seems to be redundant with the previous paragraph. I don't see how it's different from the previous paragraph. And it seems obvious to that the a process is can be passed to waitpid(), I don't see the point of saying it here.

Reply:
Really some confusion here. The difference we want to express here is in P_WAIT mode if the process is killed by others, VxWorks doesn't return "-signal" as unix-like OS does. VxWorks just returns -1 in this case.